subsys/net/ip/Kconfig

```diff
@@ -70,6 +70,39 @@ config NET_TC_RX_COUNT
 	  handled equally. In this implementation, the higher traffic class
 	  value corresponds to lower thread priority.
 
+choice
+	prompt "Priority to traffic class mapping"
+	help
+	  Select mapping to use to map network packet priorities to traffic
+	  classes.
+
+config NET_TC_MAPPING_STRICT
+	bool "Strict priority mapping"
+	help
+	  This is the recommended default priority to traffic class mapping.
+	  Use it for implementations that do not support the credit-based
+	  sharper transmission selection algorithm.
```

Comment:
@tgorochowik: Post-review: a typo sneaked in, s/sharper/shaper/. Hope you can see this and submit a patch ;-).